website/index.html

```diff
@@ -45,6 +45,14 @@ <h3 id="introduction">Introduction</h3>
   activity.title = (android.widget.TextView) activity.findViewById(2130968577);
 }</pre>
 
+            <h4 id="resource">Resource Binding</h4>
+            <p> Bind a pre-defined resources with <code>@BindBool, @BindColor, @BindDimen, @BindDrawable, @BindInt, @BindString</code>, which binds an <code>R.bool</code> ID(or your specificed type) to its corresponding field.</p>
```

Comment:
`<code>` and `</code>` should be around each `@BindWhatever` (not including the commas and spaces)